pipeline_reviews/extract.py

```diff
@@ -4,15 +4,20 @@
 from os import environ
 from urllib.parse import quote_plus
 
-import pandas as pd
+from pandas import DataFrame
 from dotenv import load_dotenv
-from psycopg2 import connect, Error
+from psycopg2 import connect
 from psycopg2.extensions import connection
 from psycopg2.extras import RealDictCursor
 import requests
 
 
-def get_review_info_for_game(game_id: int) -> dict:
+class GamesNotFound(Exception):
```

Comment:
I like this error class, but maybe give a more detailed error message: it's raised when your query fails to find any games release in the last two weeks specifically

Reply:
Had that originally but thought it was too specific but if you think it should be like that - I'll change it back :)